metadata/site_import/meta/system-objecttype-extensions.xml

```diff
@@ -469,6 +465,34 @@
                 <min-length>0</min-length>
                 <field-length>0</field-length>
             </attribute-definition>
+            <attribute-definition attribute-id="ExpressPayments_enabled">
+                <display-name xml:lang="x-default">Enable express payments</display-name>
```

Comment:
Let's make the display names for the new attributes here more descriptive (full sentences) as they will be visible "as is" in the classic config page.

Reply:
I changed those to more descriptive sentences, please check.